fails to run when using kotlin v1.1.3

```
:lintKotlinMain FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':lintKotlinMain'.
> org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment$Companion.createForProduction(Lorg/jetbrains/kotlin/com/intellij/openapi/Disposable;Lorg/jetbrains/kotlin/config/CompilerConfiguration;Ljava/util/List;)Lorg/jetbrains/kotlin/cli/jvm/compiler/KotlinCoreEnvironment;
```

and in some situation says:
```
 ./gradlew check
:lintKotlinMain FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':lintKotlinMain'.
> Could not initialize class com.github.shyiko.ktlint.core.KtLint
```

gradle.properties:
```
version=1.0
group=com.foo.example

kotlinVersion=1.1.3

junitVersion = 5.0.0-M4
junitPlatformVersion = 1.0.0-M4

ktlintVersion = 0.8.3
kotlinterPluginVersion = 1.1.0
```

build.gradle:
```
buildscript {
  dependencies {
    classpath "gradle.plugin.org.jmailen.gradle:kotlinter-gradle:$kotlinterPluginVersion"

    classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
    classpath "org.junit.platform:junit-platform-gradle-plugin:$junitPlatformVersion"
  }
}

apply plugin: 'java'
apply plugin: 'kotlin'
apply plugin: "org.jmailen.kotlinter"
apply plugin: 'org.junit.platform.gradle.plugin'

dependencies {
  compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
  compile "org.jetbrains.kotlin:kotlin-test:$kotlinVersion"

  testCompile "org.jetbrains.kotlin:kotlin-reflect:$kotlinVersion"
  testCompile "org.jetbrains.kotlin:kotlin-test-junit:$kotlinVersion"

  testCompile "org.junit.platform:junit-platform-runner:$junitPlatformVersion"
  testCompile "org.junit.platform:junit-platform-launcher:$junitPlatformVersion"
  testCompile "org.junit.platform:junit-platform-engine:$junitPlatformVersion"
  testCompile "org.junit.platform:junit-platform-console:$junitPlatformVersion"

  testCompile "org.junit.jupiter:junit-jupiter-api:$junitVersion"

  testRuntime "org.junit.jupiter:junit-jupiter-engine:$junitVersion"
}

sourceCompatibility = 1.8

junitPlatform {
  platformVersion "$junitPlatformVersion"
}

kotlinter {
  indentSize = 4
}

task wrapper(type: Wrapper) {
  gradleVersion = '3.5.1'
}
```

--- a/README.md
+++ b/README.md
@@ -10,9 +10,15 @@ Available on the Gradle Plugins Portal: https://plugins.gradle.org/plugin/org.jm
 
 ```groovy
 plugins {
-    id 'org.jmailen.kotlinter' version '1.1.0'
+    id 'org.jmailen.kotlinter' version '1.2.0'
 }
 ```
+
+### Compatibility
+
+Kotlinter 1.2.0 and later compatible with Kotlin Gradle plugins 1.1.3+
+
+Kotlinter 1.1.0 and earlier compatible with Kotlin Gradle plugins 1.1 - 1.1.2-5
 
 ### Features
 - Extends Kotlin JVM and Android projects with lint and format tasks for each `SourceSet`

--- a/build.gradle
+++ b/build.gradle
@@ -12,15 +12,15 @@ repositories {
 }
 
 dependencies {
-    compile 'com.github.shyiko:ktlint:0.8.3'
-    compileOnly 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.1.2-5'
+    compile 'com.github.shyiko:ktlint:0.9.0'
+    compileOnly 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.1.3-2'
     compileOnly 'com.android.tools.build:gradle:2.3.3'
     testCompile 'junit:junit:4.12'
-    testRuntime 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.1.2-5'
+    testRuntime 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.1.3-2'
     testRuntime 'com.android.tools.build:gradle:2.3.3'
 }
 
-version = '1.1.0'
+version = '1.2.0'
 group = 'org.jmailen.gradle'
 def pluginId = 'org.jmailen.kotlinter'
 

